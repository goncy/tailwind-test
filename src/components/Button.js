import React from "react";

const Button = ({ children }) => (
  <button className="bg-gray-300 hover:bg-gray-400 text-gray-800 font-bold py-2 px-4 rounded inline-flex items-center">
    <span>{children}</span>
  </button>
);

export default Button;
