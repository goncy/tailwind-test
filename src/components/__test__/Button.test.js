import React from "react";
import { render, cleanup } from "@testing-library/react";

import Button from "../Button";

afterEach(cleanup);

it("renders", () => {
  const { asFragment } = render(<Button>Buy</Button>);

  expect(asFragment()).toMatchSnapshot();
});
