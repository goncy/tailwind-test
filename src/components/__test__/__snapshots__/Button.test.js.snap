// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <button
    class="bg-gray-300 hover:bg-gray-400 text-gray-800 font-bold py-2 px-4 rounded inline-flex items-center"
  >
    <span>
      Buy
    </span>
  </button>
</DocumentFragment>
`;
